### PR TITLE
build: use whiskers, rename to `catppuccin-theme` & bump to `1.0`

### PR DESCRIPTION
**Pull request by nullishamy**
Adds [Whiskers](https://github.com/catppuccin/whiskers), Catppuccin's in-house templating tool, to build the themes. Instead of editing `info.rkt` directly, edit the `drracket.tera` file and then run `just build` to build the output themes. You will need Whiskers (linked above) and *optionally* [Just](https://github.com/casey/just) installed.

### CODE REVIEW

**Comment by sgoudham** [comment]
Hey :wave: @PanAeon,

Are you happy for this PR to be merged? Whiskers is being rolled out across the organisation to make it easier to update themes when/if the hex codes of the theme need changing or updating.

I see that this port is uploaded to the [drracket package registry](https://pkgs.racket-lang.org/package/drracket-catppuccin). Since this port has been merged, we usually take ownership of plugins from an organisational level to avoid relying on a single maintainer to make updates. Do you know if you would be able to transfer the package to a shared Catppuccin account?

**Comment by PanAeon** [comment]
Hi, yeah, I'm totally fine. I can just delete my package after you upload it to the package registry.

File: info.rkt [hunk 1]
```diff
@@ -7,121 +7,119 @@
...
+  (define mocha-subtext0)    #(166 173 200)
+  (define mocha-overlay2)    #(147 153 178)
+  (define mocha-overlay1)    #(127 132 156)
+  (define mocha-overlay0)    #(108 112 134)
+  (define mocha-surface2)    #(88 91 112)
+  (define mocha-surface1)    #(69 71 90)
+  (define mocha-surface0)    #(49 50 68)
+  (define mocha-base)    #(30 30 46)
+  (define mocha-mantle)    #(24 24 37)
+  (define mocha-crust)    #(17 17 27)
```
Comment (sgoudham) [inline info.rkt]: Looks like these need to end in 2 `))` instead of just 1 `)`

**Review by sgoudham — CHANGES_REQUESTED** [review]
Template needs edited as per my comment above.

**Review by sgoudham — APPROVED** [review]

**Comment by sgoudham** [comment]
Hi again @PanAeon,

I've renamed the package as part of this PR to be `catppuccin-theme` and uploaded it to the package registry.

I'd appreciate if you could confirm that you can install the new package locally and then afterwards remove the `drracket-catppuccin` package. 

Thanks for your work again on this repository!

**Comment by PanAeon** [comment]
hey @sgoudham 
I have deleted my package from the registry.
Everything works except the repo name `drracket` clashes with `drracket`. I've created an issue for this. I'm not an expert on racket so can't help with the details.

**Pull request merged**